network-api/networkapi/templates/fragments/buyersguide/primary_nav.html

```diff
@@ -1,30 +1,58 @@
 {% extends "partials/primary_nav.html" %}
 
-{% load bg_nav_tags i18n localization %}
+{% load bg_nav_tags i18n localization static %}
 
+{% block mobile-search %}
+  <div id="pni-mobile-container" class="tw-hidden large:tw-hidden tw-absolute tw-mt-[64px] tw-top-0 tw-w-full tw-left-0 tw-bg-blue-05">
+    <div class="tw-px-4 tw-flex tw-items-center">
+      <img
+        class="tw-w-4 tw-h-4 tw-mr-2"
+        src="{% static "_images/buyers-guide/pni-search.svg" %}"
+      >
+      <input
+        id="pni-mobile-bar" 
+        type="text" 
+        class="tw-w-full tw-bg-blue-05 tw-outline-none tw-py-4 tw-text-base tw-text-blue-60"
```

Comment:
```suggestion
        class="tw-w-full tw-bg-blue-05 tw-outline-none tw-py-4 tw-my-1 tw-text-base tw-text-blue-60"
```

Hey Simon! Have a quick optional suggestion, I noticed on the small-medium breakpoints, the search bar doesnt seem to be covering the category filter bar on the product catalog page:
![Screenshot 2022-08-29 at 19-29-17 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/187336148-4d3cd5ef-69fd-41d2-ac10-457c9fa2e308.png)
![Screenshot 2022-08-29 at 19-29-01 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/187336153-0284bbba-b95e-4092-980a-bf2c6c97e1e8.png)


However with the suggestion above, it adds just enough space where it covers everything without adding too much extra space to the bar

![Screenshot 2022-08-29 at 19-29-45 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/187336380-33044add-5f83-401c-834a-b8dc977a9a52.png)